mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafText.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  LeafText.swift
+//  
+//
+//  Created by Cris Messias on 06/10/23.
+//
+
+import SwiftUI
+
+//MARK: - TitleModifier
+
+public struct TitleModifier: ViewModifier {
+    @Environment(\.leafTheme) private var theme
+    public func body(content: Content) -> some View {
+        content
+            .font(theme.font.titleLarge)
+            .bold()
+            .lineSpacing(0.2)
+            .foregroundStyle(theme.color.content.primary)
+
+    }
+}
+public extension View {
+    func titleModifier() -> some View {
```

Comment:
I also think it is more readable without this `modifier` suffix
```suggestion
    func title() -> some View {
```

Then we just call:
```swift
Text("Cris rocks")
   .title()
```

Reply:
Done and dust! 🪨